packages/adminrouter/extra/src/test-harness/tests/test_generic.py

```diff
@@ -0,0 +1,478 @@
+# Copyright (C) Mesosphere, Inc. See LICENSE file for details.
```

Comment:
How easy is to see which test has failed if tests are defined in external yaml file?